code/src/index.css

```diff
@@ -1,13 +1,155 @@
+@import url('https://fonts.googleapis.com/css2?family=Varela+Round&display=swap');
+
+:root {
+	--primary: hsl(0, 0%, 20%);
+	--secondary: hsl(204, 71%, 74%);
+	--tertiary: hsl(167, 37%, 73%);
+}
+
+::selection {
+	background-color: var(--tertiary);
+}
+
 body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+	margin: 0;
+	box-sizing: border-box;
+	font-family: 'Varela Round', sans-serif;
+	-webkit-font-smoothing: antialiased;
+	-moz-osx-font-smoothing: grayscale;
+
+	background-image: linear-gradient(
+		135deg,
+		hsl(204, 72%, 74%) 0%,
+		hsl(167, 37%, 73%) 100%
+	);
+	background-attachment: fixed;
+}
+
+.container {
+	width: 80%;
+	max-width: 700px;
+	height: 70vh;
+	background: hsla(0, 0%, 100%, 0.8);
+	padding: 1rem;
+	position: relative;
+	box-shadow: hsla(0, 0%, 0%, 0.2) 0 0 20px;
+	display: flex;
+	flex-direction: column;
+	margin: 1rem auto;
+	border-radius: 4px;
+}
+
+.list-container {
+	overflow-y: scroll;
+	height: 90%;
+}
+
+h1 {
+	color: #fff;
+	font-size: 1.4rem;
+	text-align: center;
+	margin: 2rem 0 1rem;
+}
+
+.counter {
+	color: #fff;
+	display: flex;
+	justify-content: center;
+	font-size: 1rem;
+	margin: -1rem;
+}
+
+form {
+	display: flex;
+	flex-direction: row;
+	position: absolute;
+	bottom: 0;
+	height: 3rem;
+	border: none;
+	margin: 0 0 0.5rem 0;
+	width: 85%;
+}
+
+input {
+	/* flex-grow: 2; */
+	border: none;
+	font-family: 'Varela Round', sans-serif;
+	font-size: 1.2rem;
+	background: hsla(0, 1%, 35%, 0);
+	padding: 0 10px;
+
+	width: 100%;
+}
+
+input:focus {
+	outline: none;
+}
+
+.add-btn {
+	border: none;
+	background-color: white;
+	font-size: 1.5rem;
+	cursor: pointer;
+	background: hsla(0, 0%, 100%, 0);
+	color: var(--primary);
+}
+
+#task-list {
+	margin: 0.5rem 0;
+	padding: 0.5rem;
+	background: hsla(0, 0%, 100%, 0.4);
+	display: flex;
+	justify-content: space-between;
+	border-radius: 4px;
+}
+.complete {
+	text-decoration: line-through;
+	color: var(--primary);
+}
+
+.delete-btn {
+	border: none;
+	background-color: transparent;
+	cursor: pointer;
+	margin-left: auto;
+	font-size: 1rem;
+	color: var(--primary);
+}
+
+.delete-btn:hover {
+	opacity: 75%;
+	transition: 0.3s ease;
+}
+
+label {
+	padding: 5px 10px;
+	font-size: 1.1rem;
+}
+
+input[type='checkbox'] {
+	appearance: none;
+	-webkit-appearance: none;
+	position: relative;
+	background: #fff;
+	/* box-shadow: inset 0 1px 1px rgba(0, 0, 0, 0.075); */
+	border: 2px solid var(--secondary);
+	padding: 10px;
+	border-radius: 50%;
+	height: 27px;
+	width: 27px;
```

Comment:
This is such a tiny detail that it doesn't really matter (more of a fun weird thing) but the checkboxes become a little oval when the task is longer than the width of the container, even though the width is set here ? Superweird because the container for all the tasks is scrollable both horizontally and vertically.